paparazzi/paparazzi/src/main/java/app/cash/paparazzi/internal/PaparazziViewOwners.kt

```diff
@@ -0,0 +1,48 @@
+package app.cash.paparazzi.internal
+
+import android.view.View
+import androidx.lifecycle.Lifecycle
+import androidx.lifecycle.LifecycleOwner
+import androidx.lifecycle.LifecycleRegistry
+import androidx.lifecycle.ViewTreeLifecycleOwner
+import androidx.savedstate.SavedStateRegistry
+import androidx.savedstate.SavedStateRegistryController
+import androidx.savedstate.SavedStateRegistryOwner
+import androidx.savedstate.setViewTreeSavedStateRegistryOwner
+
+internal object PaparazziViewOwners {
```

Comment:
can we get rid of this and have separate owners?

Reply:
Should I inline `PaparazziViewOwners.register()` into its call site at `Paparazzi#L284`? That'll also let me move `isPresentInClasspath()` to Paparazzi.

https://github.com/cashapp/paparazzi/blob/19d11c148b10599eb8b8f6cc2570227eac19dd8c/paparazzi/paparazzi/src/main/java/app/cash/paparazzi/Paparazzi.kt#L284

Reply:
The owners are already separate btw. I am using `PaparazziViewOwners` only to aggregate them.

Reply:
Yea, I think PaparazziViewOwners can be inlined.

Reply:
Done!